### PR TITLE
Updated SDK init api

### PR DESCRIPTION
**Pull request by ashinagawa**
This commit will update to use new Vungle SDK init API.

IOS-2891

### CODE REVIEW

**Review by shiv-vungle — APPROVED** [review]
LGTM

File: Vungle/VungleRouter.m [hunk 1]
```diff
@@ -120,13 +120,29 @@ - (void)initializeSdkWithInfo:(NSDictionary *)info
 #pragma clang diagnostic ignored "-Wundeclared-selector"
         [[VungleSDK sharedSDK] performSelector:@selector(setPluginName:version:) withObject:@"mopub" withObject:[[[VungleAdapterConfiguration alloc] init] adapterVersion]];
 #pragma clang diagnostic pop
+       
+        // Get delegate instance and set kinit options
```
Comment (YueVungle) [inline Vungle/VungleRouter.m]: A typo for `kinit`, thanks ^_^

**Review by YueVungle — APPROVED** [review]

**Comment by YueVungle** [comment]
@ashinagawa There is only one typo issue, the rest looks good to me. Thanks ^_^

File: Vungle/VungleRouter.m (hunk 1 above)
Comment (ashinagawa) [reply Vungle/VungleRouter.m]: Fixed. Thanks!

**Review by crbinsf — APPROVED** [review]

**Pull request merged**